examples/javascript-module-packaging/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+// import babel from 'rollup-plugin-babel';
+
+/**
+ * Usage:
+ *   $(npm bin)/rollup -c rollup.config.js && du -h dist/bundle.js
+ */
+
+export default {
+	//input: 'browser/src/main.js',
+	input: 'src/main.sjs',
+	output: {
+		// name: 'getRoles',
+		file: 'dist/bundle.js',
+		format: 'cjs',
```

Comment:
MarkLogic 10 support [ES2015 modules](https://docs.marklogic.com/guide/getting-started/javascript#id_67185). If you’re deploying to MarkLogic 10 this should be `'esm'`.